lib/libc/syslog/lib_setlogmask.c

```diff
@@ -114,11 +114,11 @@ int setlogmask(int mask)
 	 * as interrupts.
 	 */
 
-	flags = irqsave();
+	flags = enter_critical_section();
```

Comment:
Do we have `enter_critical_section`? We were using `irqsave` until now.

Reply:
Yes it is now defined in os/include/tinyara/irq.h